doc/content/devices/the-things-uno/_index.md

```diff
@@ -0,0 +1,432 @@
+---
+title: "The Things Uno"
+description: ""
+weight: 
+---
+
+{{< figure src="TheThingsUno.png" alt="CloudGate Gateway" class="float plain" >}}
+
+The [Things Uno](https://www.thethingsnetwork.org/docs/devices/uno/) is based on the [Arduino Leonardo](https://store.arduino.cc/usa/leonardo) (not the Arduino Uno) with an added **Microchip** LoRaWAN module ([RN2483](https://www.microchip.com/wwwproducts/en/RN2483) or [RN2903](https://www.microchip.com/wwwproducts/en/RN2903)). It is fully compatible with the **Arduino IDE** and existing Arduino shields.
+
+**The Things Uno** lets you easily get started with **LoRaWAN** application development using the Arduino IDE. Choose the correct Things Uno board to suit the recommended frequency plan of your country/region.
+
+## Setting up Arduino IDE
+The Arduino Integrated Development Environment (IDE) allows you to write code using the Arduino programming language (C/C++), compile (verify), and upload it to the Things Uno board. It supports Windows, Mac OS, Linux, and Chrome OS.
+
+Download the latest release of the [Arduino IDE](https://www.arduino.cc/en/Main/Software). There are different ways to install it. The full instructions can be found at https://www.arduino.cc/en/Guide for each operating system. After you have downloaded and installed the Arduino IDE, you will need to start the IDE.
+
+In the **menu bar** select **Sketch &#8594; Include Library &#8594; Manage Libraries...** or **Tools &#8594; Manage Libraries...**
+
+In the **Library Manager** window search for **The Things Network**.
+
+Select **TheThingsNetwork** Arduino library from the search results.
+
+Select the **Install** button.
+
+When the installation finishes, close the **Library Manager** by selecting the **Close** button.
+
+{{< figure src="LibraryManager.png" alt="Library Manager" >}}
+
+## Connecting Things UNO to your Computer
+To begin, connect your Things Uno to the computer with the USB cable. 
+
+[INSERT IMAGE]
```

Comment:
These images you can just take from the TTN doc, they are not TTS specific

Reply:
no image was found for connecting the thins uno with a computer.